components/blocks/FilesToCards.vue

```diff
@@ -59,26 +61,229 @@
         </template>
       </DCard>
     </div>
+    <div v-else>
+      <div class="sort-bar">
+        <multiselect
+          class="item-a"
+          v-model="searchGroup"
+          :options="cardTags()"
+          :close-on-select="true"
+          :clear-on-select="false"
+          :preserve-search="true"
+          :multiple="true"
+          placeholder="Select tags to filter by"
+          :allow-empty="true"
+        >
+        </multiselect>
+        <button
+          class="item-b button is-normal is-warning"
+          @click="clearAll"
+        >
+          Clear All Filter Tags
+        </button>
+        <multiselect
+          class="item-c"
+          v-model="sortBy"
+          :options="sortByOptions()"
+          :close-on-select="true"
+          :clear-on-select="false"
+          :multiple="false"
+          placeholder="Sort by"
+          label="name"
+          track-by="name"
+          >\
+        </multiselect>
+        <button
+          class="item-d button is-normal is-warning"
+          @click="ascending = !ascending"
+        >
+          <i v-if="ascending" class="mdi mdi-sort-ascending"></i>
+          <i v-else class="mdi mdi-sort-descending"></i>
+        </button>
+      </div>
+      <div
+        class="
+          card-container
+          is-flex
+          mt-6
+          is-justify-content-space-evenly is-flex-wrap-wrap
+        "
+      >
+        <DCard
+          v-for="(item, i) in sortedArray"
+          :key="'help-card-' + i"
+          class="mx-3 my-3 px-3"
+          variant="light"
+          accent="link"
+          width="medium"
+        >
+          <template #header>
+            <span v-if="item.group" class="radius-0 tag is-link has-text-light"
+              ><span class="icon"><i class="mdi mdi-account-multiple" /></span
+              ><abbr :title="item.group | expandAcronym">
+                {{ item.group }}
+              </abbr></span
+            >
+            <h2>{{ item.title }}</h2>
+            <div v-if="item.date">Updated: {{ item.date }}</div>
+            <span v-if="item.authors" class="small has-text-link"
+              >Authors: {{ item.authors.map((a) => a.name).join(', ') }}</span
+            >
+          </template>
+          <template #content>
+            {{ item.description }}
+          </template>
+          <template #footer>
+            <div v-if="item.links" class="link-group">
+              <a
```

Comment:
I wonder if making this a v-for would be easier? you'd need a lookup object for the icons, but something like

```html
<a v-for="(type, link) in item.links" class="link-item d-button has-text-link" :href="link" :key="type">
  <span>{{ type.toUppercase() }}<span class="icon ml-2"><i :class="["mdi", LINK_ICONS[type]]" /></span></span>
</a>
```